build.sbt

```diff
@@ -16,6 +16,10 @@ libraryDependencies += "org.apache.spark" %% "spark-core" % sparkVersion.value %
 
 libraryDependencies +=  "org.apache.hadoop" % "hadoop-client" % "2.6.5" % "provided"
 
+libraryDependencies += "com.mashape.unirest" % "unirest-java" % "1.4.9"
```

Comment:
Is this library part of spark? If not, do users have to reference this jar in addition to sparklens jar?